mopidy/core/mixer.py

```diff
@@ -1,10 +1,24 @@
 from __future__ import absolute_import, unicode_literals
 
+import contextlib
 import logging
 
+from mopidy import exceptions
 from mopidy.utils import validation
 
 
+@contextlib.contextmanager
+def _mixer_error_handling(mixer):
+    try:
+        yield
+    except exceptions.ValidationError as e:
+        logger.error('%s mixer returned bad data: %s',
+                     mixer.actor_ref.actor_class.__name__, e)
+    except Exception:
+        logger.exception('%s mixer caused an exception.',
+                         mixer.actor_ref.actor_class.__name__)
+
+
 logger = logging.getLogger(__name__)
```

Comment:
Nitpick: In all the other files you defined the error handler *after* the logger, which also flows better since the error handler use the logger.